app/javascript/components/artists/AllArtists.jsx

```diff
@@ -120,7 +163,24 @@ class AllArtists extends React.Component {
             </div>
             <div className="col-list-3">
               {artists.slice(artistStartIndex, artistEndIndex).map((artist, i) => {
-                return <ArtistColumnPanel key={i} artist={artist} />
+                return (
+                  <div>
+                    <ArtistColumnPanel key={i} artist={artist}>
+                      {artist.hidden == false && this.props.userType == "admin" &&
+                      <Button className="ml2" type="hover-button" onClick={() => this.hideArtist(artist.id)}>
```

Comment:
Styling: leftmost button shouldn't have margin-left
```suggestion
                      <Button type="hover-button" onClick={() => this.hideArtist(artist.id)}>
```